server/src/modules/users/infra/http/routes/user-router.ts

```diff
@@ -1,5 +1,11 @@
 import { Router } from 'express'
 import { Controllers } from '../../../../../setup/application'
+import RateLimit from 'express-rate-limit'
```

Comment:
Was going to include this in separate PR, but thought it would worth it to squeeze it into this once.